### PR TITLE
RPRBLND-2079: Blender crash with AOV and Hybrid Pro

### PR DESCRIPTION
**Pull request by VascoPi**
### PURPOSE
Blender crash with AOV and Hybrid Pro.

### EFFECT OF CHANGE
Fixed crash Interactive mode with enabled unsupported AOVs.

### TECHNICAL STEPS
Overrided attach_aov method and add log message.
Overrided and improved detach_aov so Core exception if exists raise at the end of the method.
Added constant SUPPORTED_AOVS.

### CODE REVIEW

File: src/bindings/pyrpr/src/pyhybridpro.py [hunk 1]
```diff
@@ -74,6 +80,17 @@ def render(self):
         for _ in range(iterations):
             super().render()
 
+    def attach_aov(self, aov, frame_buffer):
+        if aov not in SUPPORTED_AOVS:
+            log.warn("Unsupported AOV", aov)
```
Comment (bnagirniak) [inline src/bindings/pyrpr/src/pyhybridpro.py:85]: should you use return here?

**Review by bnagirniak — APPROVED** [review]
Tested - works good.

**Review by ARozghon — APPROVED** [review]
no errors with Hybrid Pro and all active AOVs

**Comment by bsavery** [comment]
PURPOSE

Blender crash with AOV and Hybrid Pro.

EFFECT OF CHANGE

Fixed crash Interactive mode with enabled unsupported AOVs.

**Pull request merged**